website/src/categories/components/toggle-group/accessibility.md

```diff
@@ -32,8 +32,8 @@ WAI-ARIA Roles, States, and Properties for a toggle group provide essential info
 |Attribute | Value | Description                                                                                                                                                                                             | User supplied <sl-icon name="info" aria-describedby="tooltip1" size="md"></sl-icon><sl-tooltip id="tooltip1">Specifies whether the attribute is always set in the component (no) or it needs to be provided by the developer (yes)</sl-tooltip>|
 |-|-|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-|
 |`role`	|`region`| Identifies the toggle group as a significant element of the page and that it should be easily accessible with screen readers.                                                                           |no|
-|`aria-label`|string| Provide an accessible name for the toggle button in it when an there is an icon-only variant used. The `aria-label` can be also added for the toggle group to describe the purpose of the toggle group. |yes|
-|`aria-labelledby`|string| Can be used to connect with a single header/element that describes the toggle group and there is no `aria-label`.                                                                                        |yes|
+|`aria-label`|string|The `aria-label` can be added for the toggle group to describe the purpose of the toggle group. Individual button labels, especially in case of icon only buttons, need to be set on button level. More information about accessibility of toggle buttons can be found in <a href="/categories/components/toggle-button/accessibility/" target="_blank">the toggle button documentation</a>. |yes|
```

Comment:
```suggestion
|`aria-label`|string|The `aria-label` can be added on the toggle group to describe its purpose. Individual button labels, especially in case of icon only buttons, need to be set on button level. More information about accessibility of toggle buttons can be found in <a href="/categories/components/toggle-button/accessibility/" target="_blank">the toggle button documentation</a>. |yes|
```